python/lsst/consdb/hinfo.py

```diff
@@ -578,5 +591,10 @@ async def main() -> None:
 
 
 if __name__ == "__main__":
+    import sys
+
     engine = setup_postgres()
-    asyncio.run(main())
+    if len(sys.argv) > 1:
+        process_local_path(sys.argv[1])
```

Comment:
So if this is called with an argument, then the service does not run as a server but just the once, is that correct?

Reply:
Yes, that's correct.